src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs

```diff
@@ -32,26 +33,26 @@ public class CreateCompanyCommand : ICreateCompanyCommand
     private readonly ICompanyChangesRepository _companyChangesRepository;
     private readonly IHttpContextAccessor _httpContextAccessor;
 
-    private bool UpdateSmtp(SmtpInfo smtpInfo, List<string> errors)
+    private async Task<bool> UpdateSmtp(SmtpInfo smtpInfo, List<string> errors)
     {
       string message = "Can not update smtp credentials.";
 
       try
       {
-        var response = _rcUpdateSmtp.GetResponse<IOperationResult<bool>>(
+        Response<IOperationResult<bool>> response = await _rcUpdateSmtp.GetResponse<IOperationResult<bool>>(
           IUpdateSmtpCredentialsRequest.CreateObj(
             host: smtpInfo.Host,
             port: smtpInfo.Port,
             enableSsl: smtpInfo.EnableSsl,
             email: smtpInfo.Email,
-            password: smtpInfo.Password)).Result.Message;
+            password: smtpInfo.Password));
 
-        if (response.IsSuccess && response.Body)
+        if (response.Message.IsSuccess && response.Message.Body)
         {
           return true;
```

Comment:
you can remove response.Message.Body from 50 string and return it (return response.Message.Body)